src/index.html

```diff
@@ -22,6 +22,425 @@
+          alt="Apple iMac"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__text-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__text-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__text-price">Price:</p>
+          <p class="card__text-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="Apple iMac"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__text-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__text-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__text-price">Price:</p>
+          <p class="card__text-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="Apple iMac"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__text-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__text-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__text-price">Price:</p>
+          <p class="card__text-value">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
```

Comment:
The `data-qa` attribute here is set to `hover`, which is inconsistent with the first card button that uses `data-qa="card-hover"`. Ensure consistency in naming conventions for easier maintenance and understanding.